Common/src/dual_grid_structure.cpp

```diff
@@ -59,7 +59,7 @@ CPoint::CPoint(unsigned short val_nDim, unsigned long val_globalindex, CConfig *
 	Coord = new su2double[nDim];
 
 	/*--- Indicator if the control volume has been agglomerated ---*/
-  
+    Parent_CV = 0;
```

Comment:
Can you please double check that the other constructors for CPoint are ok too? There are 3 of them due to overloading, I believe.

Also, small thing, but can you please take the opportunity to clean up the indentation in this routine (and others) of this file? We can make a big impact by cleaning small pieces of the code as we go. The standard is two spaces (no tab characters).